rpc/trace_test.go

```diff
@@ -136,7 +144,8 @@ func TestSimulateTransaction(t *testing.T) {
 			test.SimulateTxnInput.Txns,
 			test.SimulateTxnInput.SimulationFlags)
 		require.NoError(t, err)
-		require.Equal(t, test.ExpectedResp.Txns, resp)
+		require.Equal(t, test.ExpectedResp.Txns[0].FeeEstimate, resp[0].FeeEstimate)
```

Comment:
We should compare the entire transaction, right?

Reply:
Yeah, but it was failing as the "state_diff" was different. I'll try your suggestion on slack